sdks/python/apache_beam/testing/analyzers/README.md

```diff
@@ -0,0 +1,132 @@
+  num_runs_in_change_point_window: 7
+```
+
+**NOTE**: `test_name` should be in the format `apache_beam.foo.bar`. It should point to a single test target.
+
+**Note**: If the source is **BigQuery**, the metrics_dataset, metrics_table, project and metric_name should match with the values defined for performance/load tests.
+The above example uses this [test configuration](https://github.com/apache/beam/blob/0a91d139dea4276dc46176c4cdcdfce210fc50c4/.test-infra/jenkins/job_InferenceBenchmarkTests_Python.groovy#L30)
+to fill up the values required to fetch the data from source.
+
+### Different ways to avoid false positive change points
+
+**min_runs_between_change_points**: As the metric data moves across the runs, the change point analysis can place the
+change point in a slightly different place. These change points refer to the same regression and are just noise.
+When we find a new change point, we will search up to the `min_runs_between_change_points` in both directions from the
+current change point. If an existing change point is found within the distance, then the current change point will be
+suppressed.
+
+**num_runs_in_change_point_window**: This defines how many runs to consider from the most recent run to be in change point window.
+Sometimes, the change point found might be way back in time and could be irrelevant. For a test, if a change point needs to be
+reported only when it was observed in the last 7 runs from the current run,
+setting `num_runs_in_change_point_window=7` will achieve it.
+
+##  Register a test for performance alerts
+
+If a new test needs to be registered for the performance alerting tool, please add the required test parameters to the
+config file.
+
+## Triage performance alert issues
+
+All the performance/load tests metrics defined at [beam/.test-infra/jenkins](https://github.com/apache/beam/tree/master/.test-infra/jenkins) are imported to [Grafana dashboards](http://104.154.241.245/d/1/getting-started?orgId=1) for visualization. Please 
+find the alerted test dashboard to find a spike in the metric values.
+
+For example, for the below configuration,
+* test: `apache_beam.testing.benchmarks.inference.pytorch_image_classification_benchmarks`
+* metric_name: `mean_load_model_latency_milli_secs`
+
+Grafana dashboard can be found at http://104.154.241.245/d/ZpS8Uf44z/python-ml-runinference-benchmarks?orgId=1&viewPanel=7
+
+If the dashboard for a test is not found, you can use the 
+code below to generate a plot for the given test, metric_name. 
+
+If the performance/load test store the results in BigQuery using this [schema](https://github.com/apache/beam/blob/83679216cce2d52dbeb7e837f06ca1d57b31d509/sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py#L66),
+then use the following code to fetch the metric_values for a `metric_name` for the last `30` runs and display a plot using matplotlib.
+
+**NOTE**: Install matplotlib and pandas using `pip install matplotlib pandas`.
```

Comment:
nice, thanks. this would be better as a notebook in the repo, that people could refer to and update as necessary.